public/static/partials/user_host_options.html

```diff
@@ -112,9 +112,9 @@
           </ul>
           <div ng-show="!$parent.homeVolumeID">
             <label> Volume size (GB) </label>
-            <input type="number" ng-disabled="availableVolumeSize() <= 0" ng-model="$parent.homeVolumeSize" min="1"/>
+            <input type="number" ng-model="$parent.homeVolumeSize"/>
             <span class="warning-text" ng-show="invalidVolumeSize($parent.homeVolumeSize)"><i class="fa fa-warning"></i>
-          Can have at most [[ maxVolumeSizePerUser ]] GiB per user
+                [[ $parent.homeVolumeSize > 0 ? availableVolumeSize() + "GiB of " + maxVolumeSizePerUser + "GiB remaining" : "Size must be greater than 0" ]]
```

Comment:
We talked earlier about including a link to the volumes modal for people to see what volumes they currently have in the case of this error, since this new table has caused confusion a few times. What do you think?